src/coprocessor/dag/expr/err.rs

```diff
@@ -62,6 +63,12 @@ quick_error! {
             description("evaluation failed")
             display("{}", s)
         }
+        Regex(err: regex::Error) {
```

Comment:
What's the error code for this error? Could we use the `Error::Eval`  instead?

Reply:
You're right, I will fix the code, thanks